src/main/org/nlogo/api/Argument.scala

```diff
@@ -155,4 +155,23 @@ trait Argument {
   @throws(classOf[LogoException])
   def getCommandTask: CommandTask
 
+  /**
+   * Returns the argument as a <code>org.nlogo.api.CommandTask</code>.
+   *
+   * @throws ExtensionException if the argument is not a <code>CommandTask</code>
```

Comment:
Give the full path to this class (`org.nlogo.api....`), since scaladoc isn't smart enough to figure out what you mean. This should be done for each of the `@throws` annotations added here.